src/internal/tester/its/resourceService.go

```diff
@@ -0,0 +1,25 @@
+package its
+
+import "github.com/alcionai/corso/src/pkg/path"
+
+type ResourceServicer interface {
+	Resource() IDs
+	Service() path.ServiceType
+}
+
+type resourceAndService struct {
+	ProtectedResource IDs
+	ServiceType       path.ServiceType
```

Comment:
could you please make these unexported fields? There's an interface for getting them so there shouldn't be a need to export them to other packages